experimenting with translate as secondary link in extended header

pglevy opened the pull request.

pglevy commented:
@bnaolu, see any product page (not home page) for extended header with secondary links for incorporating translate link/button. Just trying this out to see how it feels. What about use of icons?

bnaolu reviewed (approved):
I think that's a good starting place to put the translate link/button. Still not sure how I feel about the icons, though: without the icons, it matches the consistency of the page. With keeping the icons, it does help with drawing my eyes there and they make recognition of these links easier.

pglevy commented:
Updated the branch to remove the icons. Go with that for now? Also added the "current" indicator in the main nav. (Closes #9)

![image](https://user-images.githubusercontent.com/5177337/119498663-dafe1b00-bd33-11eb-86b5-a186e01537c4.png)
